nginx.conf

```diff
@@ -1,16 +1,17 @@
 events {
 }
 http {
-  # This path may be different on your local computer depending on how you
-  # installed nginx.
+  # This path may be different on your local computer depending on your OS and
+  # how you installed nginx.
   include /usr/local/etc/nginx/mime.types;
 
   server {
     listen 8080;
     server_name localhost;
-    # Specify the root of the repository to `nginx -p` so that . is relative to
-    # the root directory of the repository.
+    # Specify the root directory of the repository to `nginx -p` so that . is
+    # relative to the root of the repository.
     root .;
+    expires -1;
```

Comment:
perhaps leave a comment mentioning that this setting is only desirable in a development environment, and for production you'll want to remove it.

Reply:
👍